src/EvalFile_Command.php

```diff
@@ -41,22 +44,31 @@ public function __invoke( $args, $assoc_args ) {
 			WP_CLI::error( "'$file' does not exist." );
 		}
 
+		$use_include = Utils\get_flag_value( $assoc_args, 'use-include', false );
+
+		if ( '-' === $file && $use_include ) {
+				WP_CLI::error( '"-" and "--use-include" parameters cannot be used at the same time' );
+		}
+
 		if ( null === Utils\get_flag_value( $assoc_args, 'skip-wordpress' ) ) {
 			WP_CLI::get_runner()->load_wordpress();
 		}
 
-		self::execute_eval( $file, $args );
+		self::execute_eval( $file, $args, $use_include );
 	}
 
 	/**
 	 * Evaluate a provided file.
 	 *
 	 * @param string $file Filepath to execute, or - for STDIN.
 	 * @param mixed  $args Array of positional arguments to pass to the file.
+	 * @param bool $use_include Use include instead of eval given a $file Filepath to execute.
 	 */
-	private static function execute_eval( $file, $args ) {
+	private static function execute_eval( $file, $args, $use_include ) {
 		if ( '-' === $file ) {
 			eval( '?>' . file_get_contents( 'php://stdin' ) );
+		} elseif ( $use_include ) {
+				include $file;
```

Comment:
Misaligned indentation here.
```suggestion
			include $file;
```